frontend/src/Pages/Home.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+// import WbHeaderBsNav from '../components/WbHeaderBsNav';
+// import Retrieve from '../components/Retrieve';
+import PageNavHeader from '../components/PageNavHeader';
+import LiveData from '../components/LiveData';
+import './styleHome.css';
+import WhyWB from '../components/WhyWB';
+import WorkShop from '../components/WorkShop';
+import Partners from '../components/Partners';
+import Directions from '../components/Directions';
+import Community from '../components/Community';
+import Foot from '../components/Footer';
+
+class Home extends Component {
+  render() {
+    return (
+      <div>
+        <PageNavHeader />
+        {/* <WbHeaderBsNav />
+        <Retrieve /> */}
+        <Directions />
+        <LiveData />
+        <WhyWB />
+        <Community />
+        <WorkShop />
+        <Partners />
+        <Foot />
```

Comment:
Will Foot and PageNavHeader be accessible only within Home? Or universally throughout the app? If universally, we should move these components up to App.js.